conf/DynamicAddressing.h

```diff
@@ -41,7 +41,7 @@
 #define USEEEPROM							1
 #define	IPBROADCAST							1
 
-#if(MCU_TYPE == 0x01)	// Atmel AVR Atmega
+#if(MCU_TYPE == 0x01 || MCU_TYPE == 0x03 && ETH_ESP32 == 1)	// Atmel AVR Atmega or ESP32 with ethernet
```

Comment:
When using Ethernet with ESP32 we should be able to use the same networking as with Arduino. All seemed to work in my tests.